lib/actions/help.coffee

```diff
@@ -91,8 +91,24 @@ command = (params, options, done) ->
 			console.log('\nOptions:\n')
 			print(parse(command.options))
 
+		if params.command
+			listNested(params.command, command.signature)
+
 		return done()
 
+listNested = (commandName) ->
+	list = []
+
+	for command in global._capitano.state.commands
+		parameters = command?.signature?.parameters
+		parameterName = parameters?[0].parameter
+
+		if parameterName is commandName and not (/[^a-z]{1,}/g).test(parameters?[1])
+			list.push("#{parameters?[1]?.parameter}")
```

Comment:
It looks like this could be undefined, though I'm not 100% sure when... I think in that case we should probably avoid pushing anything at all.